docs/install/helm.md

```diff
@@ -151,11 +151,12 @@ The following table lists the configurable parameters of the AKO chart and their
 | `NetworkSettings.enableRHI` | Publish route information to BGP peers | false |
 | `NetworkSettings.bgpPeerLabels` | Select BGP peers using bgpPeerLabels, for selective VsVip advertisement. | `Empty List` |
 | `NetworkSettings.nsxtT1LR` | Unique ID (note: not display name) of the T1 Logical Router for Service Engine connectivity. Only applies to NSX-T cloud.| `Empty string` |
+| `NetworkSettings.defaultDomain` | Specify a default sub-domain for L4 LB services/L7 virtual service created from OpenShift route. | First domainname found in cloud's dnsprofile for L4 vs and `empty string` for L7 vs |
```

Comment:
Half part of this sentence `Specify a default ... ` points to K8s side object and other half (after `/`) points to AviController side object. Can we make it uniform? Either point it out to AviController side objects or K8s/Openshift side objects.